### ISSUE TITLE
Allow choosing TTS voice

### ISSUE THREAD

**Issue by tuomas2** [AndBible/and-bible#939]
Popular request and fixes some issues.

**Comment by agrogers** [comment]
I suggest we delay this one to after v3.4 release
I think the speech interface needs a little work. This could be grouped in with that perhaps.

**Comment by tuomas2** [comment]
Yes, not for 3.4 any more, but is this something that we should prepare strings already?

**Comment by tuomas2** [comment]
Does it need any strings? a spnner with voice choises should do? https://developer.android.com/guide/topics/ui/controls/spinner

**Comment by tuomas2** [comment]
User on Chromebook tells about issue that default speaker is used instead of configured system default. https://support.andbible.org/scp/tickets.php?id=886

**Comment by Noitarud** [comment]
Different speaker for red letter? If quotes are used assume another speaker and use a different one for these others (maybe a slower pace?)

**Comment by Noitarud** [comment]
Crazy idea https://stackoverflow.com/questions/51110887/hard-coding-the-pico-tts-engine-into-an-android-app

**Comment by meichthys** [comment]
We can already choose different TTS voices via the system menu. Is this Feature request intended to allow changing the TTS option from within AndBible (without affecting the System TTS voice)?

**Comment by tuomas2** [comment]
> We can already choose different TTS voices via the system menu. Is this Feature request intended to allow changing the TTS option from within AndBible (without affecting the System TTS voice)?

Yes. Some users do not want to change system TTS voice settings and/or it's too complicated for them.
